### PR TITLE
AAP-14045 - Incorporating crowdsourced minimalism review changes (P1)

### PR DESCRIPTION
**Pull request by dcdacosta**
This PR incorporates the improvements suggested in the crowdsourced minimalism review performed on the content. Changes are being tracked in https://issues.redhat.com/browse/AAP-14045:

Link to preview draft (VPN required): http://file.rdu.redhat.com/~ddacosta/AAP14045/master.html

### CODE REVIEW

**Review by hherbly — COMMENTED** [review]
Donna, the revisions you've made to this piece of writing have made it much more polished and easier to read--I think you incorporated the feedback from the session really well. I added a few comments to polish it some more. A couple of them point out typos and things that need to be fixed, but the majority are stylistic suggestions, so implement whichever ones you like and disregard the ones that don't work. Lmk if you want me to review again!

File: downstream/assemblies/hub/assembly-collection-import-export.adoc [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
 [id="export-import-collections"]
-= Exporting and importing collections
+= Exporting and importing collections in {HubName}
 
 :context: export-import
 
 [role="_abstract"]
-Ansible {HubName} stores collections within repositories. These collections are versioned by the curator, therefore, many versions of the same collection can exist in the same or different repositories at the same time.
+Ansible {HubName} stores automation content collections within repositories. These collections are versioned by the automation content creator, therefore, many versions of the same collection can exist in the same or different repositories at the same time.
```
[the remark this answers is not in the document]
Comment (dcdacosta) [reply downstream/assemblies/hub/assembly-collection-import-export.adoc:9]: Change implemented.

File: downstream/assemblies/hub/assembly-repo-management.adoc [hunk 2]
```diff
@@ -1,21 +1,28 @@
...
+As an automation hub administrator, you can create, edit, delete, and move automation content collections between repositories.
 
-Staging repositories:: Any user with permission to upload for a namespace can publish content into these repositories. Content in these repositories are not available in the search page, but rather, are displayed on the approval dashboard for an administrator to verify.
+{HubNameStart} includes two types of repositories where you can publish collections:
 
-Custom repositories:: Any user with write permissions on the repository can publish content to these repositories. These repositories are not displayed on the approval dashboard. They can show up on the search page depending on whether search is enabled by the owner of the repository. Custom repositories can be private repositories where only users with view permissions can see them, or public where all users can see them.
+Staging repositories:: Any user with permission to upload for a namespace can publish collections into these repositories. Collections in these repositories are not available in the search page, but rather, are displayed on the approval dashboard for an administrator to verify.
```
[the remark this answers is not in the document]
Comment (dcdacosta) [reply downstream/assemblies/hub/assembly-repo-management.adoc:13]: Change implmented.

File: downstream/assemblies/hub/assembly-repo-management.adoc [hunk 3]
```diff
@@ -1,21 +1,28 @@
...
+
+Custom repositories:: Any user with write permissions on the repository can publish collections to these repositories. These repositories are not displayed on the approval dashboard. They can show up on the search page depending on whether search is enabled by the owner of the repository. Custom repositories can be private repositories where only users with view permissions can see them, or public where all users can see them.
```
[the remark this answers is not in the document]
Comment (dcdacosta) [reply downstream/assemblies/hub/assembly-repo-management.adoc:15]: Suggested change implemented.

File: downstream/modules/automation-hub/proc-adding-collections-repository.adoc [hunk 4]
```diff
@@ -3,9 +3,9 @@
 
 [id="proc-adding-collections-repository"]
 
-= Adding collections to a repository
+= Adding collections to a {HubName} repository
 
-After your repository is created, you can begin adding collections to it.
+After your repository is created, you can begin adding automation content collections to it.
```
[the remark this answers is not in the document]
Comment (dcdacosta) [reply downstream/modules/automation-hub/proc-adding-collections-repository.adoc:8]: Change implemented.

File: downstream/modules/automation-hub/proc-create-repository.adoc [hunk 5]
```diff
@@ -17,20 +17,22 @@ Use this procedure to create a repository.
 +
 [NOTE]
 ====
-If there are issues with a change to your custom repository, retained versions allow you to xref:proc-revert-repository-version[revert to a different repository version].
+If you have a problem with a change to your custom repository, you xref:proc-revert-repository-version[revert to a different repository version] that you have retained.
```
[the remark this answers is not in the document]
Comment (dcdacosta) [reply downstream/modules/automation-hub/proc-create-repository.adoc:20]: Good catch! Change implemented.

File: downstream/modules/automation-hub/proc-create-repository.adoc [hunk 6]
```diff
@@ -17,20 +17,22 @@ Use this procedure to create a repository.
...
 ====
 +
 . Select the *Pipeline* for the repository. The available options include the following:
```
[the remark this answers is not in the document]
Comment (dcdacosta) [reply downstream/modules/automation-hub/proc-create-repository.adoc:23]: Change implemented.

File: downstream/modules/automation-hub/proc-export-collection.adoc [hunk 7]
```diff
@@ -3,9 +3,9 @@
 
 [id="proc-export-collection"]
 
-= Export a collection
+= Exporting an automation content collection in {HubName}
 
-After collections are finalized, they can be delivered to new destinations and distributed to  others users across your organization. Use the following procedure to export a collection.
+After collections are finalized, you can deliver them to new destinations for distribution to others user across your organization.
```
[the remark this answers is not in the document]
Comment (dcdacosta) [reply downstream/modules/automation-hub/proc-export-collection.adoc:8]: @hherbly I believe an organization can have more than one automation hub, so collections can be moved/shared between hub instances, between repositories and among users. I'll use your second suggested sentence for better clarity.

File: downstream/modules/automation-hub/con-approval-pipeline.adoc [hunk 8]
```diff
@@ -4,15 +4,15 @@
...
 
 Auto approval::
 When auto approve is enabled, any collection uploaded to a staging repository is automatically promoted to all of the repositories marked as `pipeline=approved`.
 
 Approval required::
-When an administrator navigates to the approval dashboard, they can see the collections that have been uploaded into any of the staging repositories. From here they are presented with an Approve button. Clicking btn:[Approve] displays a list of approved repositories. From this list, the administrator can select 1 or more repositories to which the content should be promoted.
+When an administrator navigates to the approval dashboard, they can see the collections that have been uploaded into any of the staging repositories. From here they are presented with an Approve button. Clicking btn:[Approve] displays a list of approved repositories. From this list, the administrator can select one or more repositories to which the content should be promoted.
 +
 If only one approved repository exists, the collection is automatically moved into it and the administrator is not prompted to select a repository.
```
[the remark this answers is not in the document]
Comment (dcdacosta) [reply downstream/modules/automation-hub/con-approval-pipeline.adoc:17]: Change implemented.

File: downstream/modules/automation-hub/proc-revert-repository-version.adoc [hunk 9]
```diff
@@ -3,9 +3,9 @@
 
 [id="proc-revert-repository-version"]
 
-= Revert to a different repository version
+= Revert to a different {HubName} repository version
 
-When content is added or removed from a repository, a new version of it is created. If there are issues with a change to your custom repository, you can revert to a previous version. Reverting is a safe operation and does not delete content from the system, but rather, changes the content associated with the repository. The number of versions saved is defined in the *Retained number of versions* setting when a xref:proc-create-repository[repository is created].
+When automation content collections are added or removed from a repository, a new version of it is created. If there are issues with a change to your repository, you can revert to a previous version. Reverting is a safe operation and does not delete collections from the system, but rather, changes the content associated with the repository. The number of versions saved is defined in the *Retained number of versions* setting when a xref:proc-create-repository[repository is created].
```
[the remark this answers is not in the document]
Comment (dcdacosta) [reply downstream/modules/automation-hub/proc-revert-repository-version.adoc:8]: Verbosity strikes again! Change implemented.

**Review by hherbly — APPROVED** [review]
LGTM!

**Pull request merged**